avaframe/com1DFA/com1DFA.py

```diff
@@ -109,11 +108,21 @@ def com1DFAMain(avalancheDir, cfgMain, cfgFile=''):
     """
 
     modName = 'com1DFA'
+
+    # read initial configuration
+    cfgStart = cfgUtils.getModuleConfig(com1DFA, fileOverride=cfgFile, toPrint=False)
+    # check if directory with remeshed DEMs shall be cleaned
+    if cfgStart['GENERAL'].getboolean('cleanDEMremeshed') is True:
+         cleanDEMremeshed = pathlib.Path(avalancheDir, 'Inputs')
+    else:
+        cleanDEMremeshed = None
+
     # Create output and work directories
-    workDir, outDir = inDirs.initialiseRunDirs(avalancheDir, modName)
+    workDir, outDir = inDirs.initialiseRunDirs(avalancheDir, modName,
+        cleanDEMremeshed=cleanDEMremeshed)
```

Comment:
I would give the true or false to initialiseRunDirs and in initialiseRunDirs read the pathlib.Path(avalancheDir, 'Inputs') and clean if cleanDEMremeshed is True.
See the other comment in initialiseRunDirs